api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -563,13 +563,20 @@ class Config:
 )
 
 
+class OfferName(pydantic_v1.ConstrainedStr):
+    min_length = 1
+    max_length = 140
+
+
 class ProductOfferEdition(OfferEditionBase):
     offer_id: int
     category_related_fields: product_category_edition_fields | None = pydantic_v1.Field(
         None,
         description="To override category related fields, the category must be specified, even if it cannot be changed. Other category related fields may be left undefined to keep their current value.",
     )
     stock: StockEdition | None = STOCK_EDITION_FIELD
+    name: OfferName | None
+    description: str | None
```

Comment:
Ici il faut que tu migres `OFFER_DESCRIPTION_FIELD` de `api/src/pcapi/routes/public/individual_offers/v1/serialization.py` dans `api/src/pcapi/routes/public/documentation_constants/fields.py` (en dessous de `OFFER_NAME`)

Puis que tu ajoutes
```suggestion
    name: OfferName | None = fields.OFFER_NAME
    description: str | None = fields.OFFER_DESCRIPTION
```